redirects.yaml

```diff
@@ -54,3 +54,45 @@
 /docs/utilities/visualise-font-metrics: /docs/utilities/font-metrics
 /docs/utilities/no-margin: /docs/utilities/margin-collapse
 /docs/utilities/no-padding: /docs/utilities/margin-collapse
+/docs/examples/base/code-block: /docs/examples/base/code/block
```

Comment:
Could we add a bit spacing and title about those, and mention that these are redirects for the removed examples?
Just to seperate them from teh "class based redirects section above it